tokio/src/runtime/builder.rs

```diff
@@ -411,13 +466,19 @@ cfg_rt_threaded! {
 
             let clock = time::create_clock();
 
+            let (core_threads, blocking_threads) = if self.core_threads > self.max_threads {
```

Comment:
You don't need this logic as the thread_pool uses the blocking_pool to spawn threads.